migration-cstor-mayastor/replicateddb-restore.md

```diff
@@ -0,0 +1,506 @@
+ServiceAccount/velero: attempting to create resource
+ServiceAccount/velero: attempting to create resource client
+ServiceAccount/velero: created
+Secret/cloud-credentials: attempting to create resource
+Secret/cloud-credentials: attempting to create resource client
+Secret/cloud-credentials: created
+BackupStorageLocation/default: attempting to create resource
+BackupStorageLocation/default: attempting to create resource client
+BackupStorageLocation/default: created
+VolumeSnapshotLocation/default: attempting to create resource
+VolumeSnapshotLocation/default: attempting to create resource client
+VolumeSnapshotLocation/default: created
+Deployment/velero: attempting to create resource
+Deployment/velero: attempting to create resource client
+Deployment/velero: created
+DaemonSet/node-agent: attempting to create resource
+DaemonSet/node-agent: attempting to create resource client
+DaemonSet/node-agent: created
+Velero is installed! ⛵ Use 'kubectl logs deployment/velero -n velero' to view the status.
+thulasiraman_ilangovan@cloudshell:~$ 
+```
+{% endtab %}
+{% endtabs %}
+
+
+
+## Step 2: Verify Backup Availability
+
+Check the availability of your previously-saved backups. If the credentials or bucket information doesn't match, you won't be able to see the backups:
+
+
+{% tabs %}
+{% tab title="Command" %}
+```text
+velero get backup | grep 13-09-23
+```
+{% endtab %}
+
+{% tab title="Output" %}
+```text
+mongo-backup-13-09-23     Completed   0        0          2023-09-13 13:15:32 +0000 UTC   29d       default            <none>
+```
+{% endtab %}
+{% endtabs %}
+
+{% tabs %}
+{% tab title="Command" %}
+```text
+kubectl get backupstoragelocation -n velero
+```
+{% endtab %}
+
+{% tab title="Output" %}
+```text
+NAME      PHASE       LAST VALIDATED   AGE     DEFAULT
+default   Available   23s              3m32s   true
+```
+{% endtab %}
+{% endtabs %}
+
+
+
+## Step 3: Restore Using Velero CLI
+
+Initiate the restore process using Velero CLI with the following command:
+
+{% tabs %}
+{% tab title="Command" %}
+```text
+velero restore create mongo-restore-13-09-23 --from-backup mongo-backup-13-09-23
+```
+{% endtab %}
+
+{% tab title="Output" %}
+```text
+Restore request "mongo-restore-13-09-23" submitted successfully.
+Run `velero restore describe mongo-restore-13-09-23` or `velero restore logs mongo-restore-13-09-23` for more details.
+```
+{% endtab %}
+{% endtabs %}
+
+
+
+
+## Step 4: Check Restore Status
+
+You can check the status of the restore process by using the `velero get restore` command. 
+
+```
+velero get restore
+```
+
+When Velero performs a restore, it deploys an init container within the application pod, responsible for restoring the volume. Initially, the restore status will be `InProgress`. 
+
+{% hint style="note" %}
+If your storage class was originally set to `cstor-csi-disk` because you imported this PVC from a cStor volume, the status might temporarily stay as `InProgress` until the process is completed.
```

Comment:
Please change it as below:

Your storage class was originally set to `cstor-csi-disk` because you imported this PVC from a cStor volume, the status might temporarily stay as `InProgress` and your PVC will be in `pending` status